src/etc/inc/dyndns.class

```diff
@@ -1814,6 +1831,32 @@
 					}
 					break;
 				// Providers in an alphabetical order. Add code for new providers below in a correct position.
+				case 'dyfi':
+					// see specification at https://www.dy.fi/page/specification
+					if (preg_match('/badauth/i', $data)) {
+						$status = $status_intro . $error_str . gettext("User Authorization Failed");
+					} else if (preg_match('/nohost/i', $data)) {
+						$status = $status_intro . $error_str . gettext("No such host");
+					} else if (preg_match('/notfqdn/i', $data)) {
+						$status = $status_intro . $error_str . gettext("Not A FQDN!");
+					} else if (preg_match('/badip/i', $data)) {
+						$status = $status_intro . $error_str . gettext("Invalid IP. IP Address submitted is improperly formatted or is a private IP address or is on a blacklist.");
```

Comment:
I wanted to use an existing string, so there wouldn't be a new translation. However, this error happens also when the client IP is not located in Finland based on geo-ip database. Technically it's a blacklist, but mentioning the country could be better for the user?